python/lsst/fgcmcal/fgcmBuildStarsTable.py

```diff
@@ -558,17 +561,28 @@ def fgcmMakeAllStarObservations(self, groupedDataRefs, visitCat,
 
         return fullCatalog
 
-    def _get_sourceTable_visit_columns(self):
+    def _get_sourceTable_visit_columns(self, inColumns):
         """
         Get the sourceTable_visit columns from the config.
 
+        Parameters
+        ----------
+        inColumns : `list`
+            List of columns available in the sourceTable_visit
+
         Returns
         -------
         columns : `list`
-           List of columns to read from sourceTable_visit
+            List of columns to read from sourceTable_visit.
+        detectorColumn : `str`
+            Name of the detector column.
         """
-        # These "visit" and "ccd" names in the parquet tables are hard-coded.
```

Comment:
Maybe note that the detector vs ccd name is a gen2 vs gen3 thing here, otherwise it's only implied by L455 and easy to miss.